ExportScripts/export_standalone_monolithic_windows.py

```diff
@@ -258,28 +262,33 @@ def process_command(args: list,
+    game_bundle_path = args.project_path / 'AssetBundling' / 'Bundles' / 'game_pc.pak'
     process_command([asset_bundler_batch_path, 'bundles', '--assetListFile', game_asset_list_path, '--outputBundlePath', game_bundle_path, '--project-path', args.project_path, '--allowOverwrites'], cwd=args.engine_path)
 
+    # Create Launcher Layout Directory
+    import shutil
+    output_cache_path = args.output_path / 'Cache' / 'pc' 
+    output_aws_gem_path = args.output_path / 'Gems' / 'AWSCore'
+    os.makedirs(output_cache_path, exist_ok=True)
+    os.makedirs(output_aws_gem_path, exist_ok=True)
+
+    for file in glob.glob(str(pathlib.PurePath(args.project_path / 'AssetBundling' / 'Bundles' / '*.pak'))):
+        shutil.copy(file, output_cache_path)
+    for file in glob.glob(str(pathlib.PurePath(mono_build_path / 'bin' / args.config / '*.*'))):
+        shutil.copy(file, args.output_path)
+    
+    for file in glob.glob(str(pathlib.PurePath(mono_build_path / 'bin' / args.config / 'Gems' / 'AWSCore' / '*.*'))):
+        shutil.copy(file, output_aws_gem_path)
+    for file in glob.glob(str(pathlib.PurePath(args.project_path / 'launch_*.*'))):
+        shutil.copy(file, args.output_path)
 
-    #Create Launcher Zip File
-    os.makedirs( args.output_path / 'Cache' / 'pc', exist_ok=True)
-    os.makedirs( args.output_path / 'Gems' / 'AWSCore', exist_ok=True)
-
-    process_command(['xcopy', '/Y', pathlib.PurePath(args.project_path / 'AssetBundling' / 'Bundles' / '*.pak'),
-                            args.output_path / 'Cache' / 'pc'], cwd=args.engine_path)
-    process_command(['xcopy', '/Y', pathlib.PurePath(mono_build_path / 'bin' / 'profile' / '*.*'),
-                            args.output_path], cwd=args.engine_path)
-    process_command(['xcopy', '/Y', pathlib.PurePath(mono_build_path  / 'bin' / 'profile' / 'Gems' / 'AWSCore' / '*.*'),
-                            args.output_path / 'Gems' / 'AWSCore'], cwd=args.engine_path)
-    process_command(['xcopy', '/Y', pathlib.PurePath(args.project_path / 'launch_*.*'), args.output_path], cwd=args.engine_path)
-
-    if args.zip_output:
-        import shutil
+    # Optionally zip the layout directory if the user requests
+    if args.archive_output:
         archive_name = args.output_path
-        print("Zipping output directory (this may take a while)...")
-        shutil.make_archive(args.output_path, 'zip', root_dir = args.output_path)
+        logger.info("Archiving output directory (this may take a while)...")
+        shutil.make_archive(args.output_path, args.archive_output_format, root_dir = args.output_path)
 
-    print(f"Exporting project is complete! Release Directory can be found at {args.output_path}")
+    logger.info(f"Exporting project is complete! Release Directory can be found at {args.output_path}")
 
-    process_command(['explorer', args.output_path])
+    if args.quiet:
+        process_command(['explorer', args.output_path])
```

Comment:
This check should be inverted to check if its not quiet
```suggestion
    if not args.quiet:
        process_command(['explorer', args.output_path])
```